app/controllers/v0/claims_controller.rb

```diff
@@ -4,13 +4,18 @@ class ClaimsController < ApplicationController
     skip_before_action :authenticate
 
     def index
-      render json: current_user.claims,
+      render json: Claim.fetch_all(current_user.vaafi_headers),
```

Comment:
I may have just talked myself out of thinking it's a good idea. Convenience in the controller vs. separation of concerns for the models…